package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-cli",
-  "version": "1.0.0-beta.11-webpack.9",
+  "version": "1.0.0-beta.11-webpack.9-2",
```

Comment:
I am sure there is a good reason for this, and I really appreciate the efforts in making this whole thing work. But at some point this is going to draw derision from the community. Look at this version number, all the numbers in it. In some sense you could say we're calling this version 1.0.0.11.9.2.